src/createTokenBridge-ethers.ts

```diff
@@ -221,7 +246,10 @@ async function getEstimateForDeployingContracts(
   );
 
   return {
-    maxGas: gasEstimateToDeployContracts,
+    // there's already a 300% increase buffer in the SDK
+    // https://github.com/OffchainLabs/arbitrum-sdk/blob/main/src/lib/message/ParentToChildMessageGasEstimator.ts#L27
+    maxSubmissionCost,
```

Comment:
now calculating the submission cost for contracts instead of basing it on submission cost for factory